CMakeLists.txt

```diff
@@ -160,6 +164,23 @@ set(UTILS_SRC
   "${HASH_CONTAINERS_DIR}/hash8019_stack.c"
 )
 
+if(${PCURL_SBOX_UNWIND_8})
+  add_definitions("-DPCURL_SBOX_UNWIND_8")
+elseif(${PCURL_SBOX_UNWIND_4})
+  add_definitions("-DPCURL_SBOX_UNWIND_4")
+elseif(${PCURL_SBOX_UNWIND_2})
+  add_definitions("-DPCURL_SBOX_UNWIND_2")
+else(${PCURL_SBOX_UNWIND_8})
```

Comment:
```suggestion
else()
```

Reply:
btw, you have set the default to unwind 2 loops at line 28.
the else case will never be triggered.